waku/v2/README.md

```diff
@@ -134,3 +134,23 @@ You can change this to `wakunode2`, the Waku v2 node like this:
 make docker-image MAKE_TARGET=wakunode2
 docker run --rm -it statusteam/nim-waku:latest --help
 ```
+## Using Websocket and secure Websockets
+
+Websocket config needs to be enabled , by default the port is 8000 for websockets.
```

Comment:
```suggestion
Websocket support is hidden under a feature flag and must be explicitly enabled in order to get Websockets support. The default port is 8000.
```

Reply:
resolved in 765a24e1549b18153b2313c61e305eddaacd2350